examples/ibc/ics_test.go

```diff
@@ -82,6 +88,54 @@ func TestICS(t *testing.T) {
 	})
 	require.NoError(t, err, "failed to build interchain")
 
-	err = testutil.WaitForBlocks(ctx, 10, provider, consumer)
+	// err = testutil.WaitForBlocks(ctx, 5, provider, consumer)
+	// require.NoError(t, err, "failed to wait for blocks")
+
+	// get provider node
+	stakingVals, err := provider.StakingQueryValidators(ctx, stakingttypes.BondStatusBonded)
+	require.NoError(t, err)
+	valOne := stakingVals[0]
+	denom := provider.Config().Denom
+
+	// delegate from the "validator" key
+	err = provider.GetNode().StakingDelegate(ctx, "validator", valOne.OperatorAddress, fmt.Sprintf("100%s", denom))
+	require.NoError(t, err, "failed to delegate from validator")
+
+	// now wait for the relayer event to be processes
+
+	// relayer flush
+	channels, err := r.GetChannels(ctx, eRep, providerChainID)
+	require.NoError(t, err)
+	// require.Len(t, channels, 2)
+
+	// get the ics20-1 transfer channel
+	var transferChannel = ""
+	for _, channel := range channels {
+		fmt.Printf("channel: %v\n", channel)
+		// TODO: this is stuck in TRYOPEN state? channel: {STATE_TRYOPEN ORDER_UNORDERED {transfer channel-1} [connection-0] ics20-1 transfer channel-1}
```

Comment:
yeah i noticed this also. i guess when you create the initial channels for ICS the module also initiates the channel handshake for an ICS-20 channel but it deliberately leaves it in the `TRYOPEN` state. you should be able to start the relayer and it will finish the channel handshake

Reply:
Alright I got the provider update to come through, but I still can't get the transfer channel to OPEN.
 
I attempted the stop start but it does not change it. Neither did CreateChannel, Connections, Client, etc. Is there some manual rly command which pushes it over to OPEN? maybe it's meant to stay in TRYOPEN for something with ICS? not sure atm. Will look into tomorrow

Reply:
THANK YOU !!! Turns out you have to StopRelayer then StartRelayer & it picks it up. I am sure I could also change the flush time to be like 5s for testing but meh

Going to write a Helper to setup provider ICS with this and will be good to go today. Phew